templates/committee_designation/votes_list.html.twig

```diff
@@ -24,9 +24,9 @@
 
                 <tr>
                     <td>{{ vote.voterKey }}</td>
-                    {% for pool_title in ['Femme', 'Homme'] %}
+                    {% for pool_code in ['female', 'male'] %}
```

Comment:
Could we use `constant('App\ValueObject\Genders::*')` in all twig files?

Reply:
@renanbr yes, fixed 👍 
```twig
{% for pool_code in constant('App\\ValueObject\\Genders::MALE_FEMALE') %}
```